tests/monotouch-test/Foundation/NSExpressionTest.cs

```diff
@@ -45,13 +45,22 @@ void TestProperties (NSExpression expression, List<string> availableProperties)
 			}
 		}
 
-
-		[Test]
-		public void FromConstant ()
-		{
-			using (var expression = NSExpression.FromConstant (new NSString ("Foo")))
-			using (var result = expression.EvaluateWith (null, null) as NSString)
-				Assert.AreEqual ("Foo", result.ToString ());
+		[TestCase("Foo", Result = "Foo")]
+		[TestCase(null, Result = null)]
+		public object FromConstant (object input)
+		{
+            NSObject value = null;
+
+            switch (input)
+            {
```

Comment:
Coding style: opening brace goes on the same line as the switch keyword (with a space to separate the closing parenthesis and the opening brace).